gent_base.theme

```diff
@@ -733,3 +734,56 @@ function gent_base_preprocess_pager(&$variables) {
   $variables['items']['first']['text'] = '1';
   $variables['items']['last']['text'] = $total;
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function gent_base_preprocess_paragraph__call_to_action(&$variables) {
+  $variables['primary_document_size'] = format_size(_gent_base_get_entity_reference_file_size($variables, 'field_primary_action_document'));
+  $variables['secondary_document_size'] = format_size(_gent_base_get_entity_reference_file_size($variables, 'field_secondary_action_document'));
+}
+
+/**
+ * Helper function to get the file size of a referenced File entity.
```

Comment:
There should be a helper file for these.

Reply:
Fixed.